server/src/main/java/org/opensearch/indices/IndicesRequestCache.java

```diff
@@ -107,6 +108,10 @@ public final class IndicesRequestCache implements RemovalListener<IndicesRequest
      * A setting to enable or disable request caching on an index level. Its dynamic by default
      * since we are checking on the cluster state IndexMetadata always.
      */
+
+    public static final String INDICES_REQUEST_CACHE_STALENESS_THRESHOLD_SETTING_KEY = "indices.requests.cache.cleanup.staleness_threshold";
+    public static final String INDICES_REQUEST_CACHE_CLEAN_INTERVAL_SETTING_KEY = "indices.requests.cache.cleanup.interval";
```

Comment:
the only reason I'd still prefer `cleanup` over `clean` is because `cleanup` is used in all of the relevant contexts, like cleanupKey, IndicesRequestCacheCleanupManager, cleanupKeyToCountMap and all relevant methods.

Reply:
Sorry my bad, I thought you were proposing to change "indices.requests.cache.cleanup.interval" to "indices.requests.cache.clean.interval"

But I see that you were actually referring to INDICES_REQUEST_CACHE_CLEAN_INTERVAL_SETTING_KEY. 

Which I totally agree with !